openhands/runtime/plugins/agent_skills/file_ops/file_ops.py

```diff
@@ -526,17 +557,18 @@ def subtract_strings(str1, str2) -> str:
                 ret_str += lint_error + '\n'
 
                 editor_lines = n_added_lines + 20
-
-                ret_str += '[This is how your edit would have looked if applied]\n'
-                ret_str += '-------------------------------------------------\n'
+                sep = '-------------------------------------------------\n'
```

Comment:
nit: We could make a global variable `SEP` so we can re-use it in other function, but it is purely a style choice -- take it with a grain of salt

Reply:
I didn't find much more use of that except within the `test_agent_skill.py`